app/code/Magento/Catalog/Test/Mftf/ActionGroup/SaveProductAttributeWithHtmlTagsInLabelActionGroup.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="SaveProductAttributeWithHtmlTagsInLabelActionGroup">
```

Comment:
Could you please split this Action group into 2 separated ones, trying to make Action Groups a much atomic as possible?
1. `AdminSaveProductAttributeActionGroup` - so it may be used later in other validation scopes
2. `AssertSeeProductAttributeValidationErrorActionGroup` - will validate the presence of the error.

Please check the [Best Practices](https://github.com/magento/magento-functional-tests-migration/wiki/Best-practices).